lib/tasks/krikri.rake

```diff
@@ -8,38 +10,73 @@ require 'krikri/index_service'
 
 namespace :krikri do
 
-  # Tasks will execute in spec/internal unless directory is changed
-  desc 'Index a valid sample aggregation'
-  task :index_sample_aggregation do
-    agg = build(:aggregation)
+  # Note: The content of the sample original record does not actually
+  # correspond to the content of the sample aggregation.
+  desc 'Save a sample record to Marmotta and Solr'
+  task :save_sample_record => :environment do
+    original_record = build(:oai_dc_record)
+    original_record.save unless original_record.exists?
 
-    graph = agg.to_jsonld['@graph'][0]
+    # Make a DPLA::MAP::Aggregation with an original record instantiated as an
+    # ActiveTriples::Resource object.
+    agg = build(:aggregation,
+                :originalRecord => ActiveTriples::Resource
+                                .new(original_record.rdf_subject)
+          )
+
+    agg.mint_id!('krikri_sample')
```

Comment:
I'd love to see this as a custom build strategy.  It's repeated below, and I think it would be useful in other tests.

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#custom-strategies

Alternatively, it occurs to me that this pattern of instantiating `Aggregation`s, then calling 'mint_id!` might deserve it's own builder method (or object?). Thoughts?